sql/connect/server/src/main/scala/org/apache/spark/sql/connect/execution/ExecuteGrpcResponseSender.scala

```diff
@@ -53,7 +53,7 @@ private[connect] class ExecuteGrpcResponseSender[T <: Message](
   private var interrupted = false
 
   // Time at which this sender should finish if the response stream is not finished by then.
-  private var deadlineTimeMillis = Long.MaxValue
```

Comment:
Although this is private, is this safe in terms of the compatibility across multiple Spark versions?

As you know, we have a compatibility test between Apache Spark 4.0.0 and 3.5.x like the following. 

There is no compatibility issue with Apache Spark 3.5.x with this change?
- https://github.com/apache/spark/actions/workflows/build_python_connect35.yml

Reply:
There should be no compatibility issues between the 3.5 client because the only public method that allows (indirect) access to this variable is SparkEnv.get.conf.get(CONNECT_EXECUTE_REATTACHABLE_SENDER_MAX_STREAM_DURATION), and the value is correctly converted into nanoseconds at line 202.

By the way, is there any way to trigger https://github.com/apache/spark/actions/workflows/build_python_connect35.yml for this PR?